src/resources/token.js

```diff
@@ -54,6 +58,62 @@ class Token extends ResourceBase {
     await this.getContract()
     return await this.contract.methods.paused().call()
   }
+
+  /**
+   * Relays calls to the contract service
+   */
+  async call(methodName, args, opts) {
```

Comment:
Can this not use [`contractService.call`](https://github.com/OriginProtocol/origin-js/blob/john/boost_approval/src/services/contract-service.js#L187)?